stratum/chassis_config.pb_with_PortShaping.txt

```diff
@@ -0,0 +1,79 @@
+# SPDX-FileCopyrightText: Copyright 2020-present Open Networking Foundation.
```

Comment:
rename to `chassis_config_with_port_shaping.pb.txt`

Reply:
thanks for your suggestion. Done.

Reply:
still incorrect, can you update it again?

Reply:
It looks fixed to me

Reply:
> still incorrect, can you update it again?

yes , it was not correctly updated. I already fixed the file name now.